src/models/notFoundError.ts

```diff
@@ -0,0 +1,9 @@
+import { BaseErrorModel } from './baseError';
+import { NotFoundInternal } from '../types';
+
+export class NotFoundErrorModel extends BaseErrorModel {
+  public message(key: string, value: string): NotFoundInternal {
```

Comment:
Found msyelf copy-pasting code, so I made a more specific implementation of the BaseError class